packages/@contentful--app-scripts/lib/cache-credential/index.js

```diff
@@ -0,0 +1,89 @@
+const dotenv = require('dotenv');
+const fs = require('fs');
+const { EOL } = require('os');
+const chalk = require('chalk');
+
+
+const fsPromises = fs.promises;
+const fsConstants = fs.constants;
+const envFileName = '.env';
+
+// Always export set env vars by default
+dotenv.config();
+
+async function removeOldEnv(envKey) {
+  const envFileData = await fsPromises.readFile(envFileName, {encoding: 'utf-8',});
+  const envVarRegex = new RegExp(`(${EOL}|^)${envKey}=.+(${EOL})?`); // find line with envVar defined
+
+  if (envVarRegex.test(envFileData)) {
+    await fsPromises.writeFile(envFileName, envFileData.replace(envVarRegex, ''), {
+      encoding: 'utf-8'
+    });
+  }
+}
+
+async function addEnvFileToGitIgnore() {
+  try {
+    const gitIgnoreFile = '.gitignore';
+    await fsPromises.access(gitIgnoreFile, fs.constants.F_OK);
+    const gitIgnoreFileData = await fsPromises.readFile(gitIgnoreFile, {encoding: 'utf-8'});
+    const envFileNameRegex = new RegExp(`(^|${EOL})?${envFileName}*?${EOL}?`);
+
+    if (!envFileNameRegex.test(gitIgnoreFileData)) {
+      await fsPromises.writeFile(gitIgnoreFile, `${EOL}${envFileName}${EOL}`, {
+        encoding: 'utf-8',
+        flag: 'a'
+      });
+    }
+
+  } catch(err) {
+    console.log(`${
+      chalk.yellow('Warning: could not add .env file to .gitignore. Please don\'t forget to add it manually')
+    }`)
+  }
+}
+
+async function checkFileEnvExists() {
+  let envFileExists = false;
+  try {
+    await fsPromises.access(envFileName, fsConstants.F_OK);
+    envFileExists = true;
+  } catch(_) {
+    envFileExists = false;
+  }
+  return envFileExists;
```

Comment:
It's a little complex, but can be easily simplified:

```suggestion
  try {
    await fsPromises.access(envFileName, fsConstants.F_OK);
    return true;
  } catch(_) {
    return false;
  }
```